hack/make-rules/update.sh

```diff
@@ -32,10 +32,6 @@ fi
 
 SILENT=${SILENT:-true}
 ALL=${FORCE_ALL:-false}
-V=""
-if [[ "${SILENT}" != "true" ]]; then
-	V="-v"
-fi
 
```

Comment:
```
In ./hack/make-rules/update.sh line 37:
        V="-v"★
        ^-- SC2034: V appears unused. Verify use (or export if used externally).
```
In #74877,  `godep-restore.sh" ${V}` was deleted and I could not find scripts which use "V". So I deleted these lines.